testdocker/Dockerfile

```diff
@@ -41,7 +41,7 @@ RUN chmod a+x -R /usr/bin/dirdiff
 
 RUN addgroup --gid 150 intershop && \
     useradd -m --uid ${USERID} -g intershop --shell /bin/sh ${USERNAME} && \
-    useradd --uid 151 -g intershop --shell /bin/sh intershop
+    useradd --uid 151 -g intershop --shell /bin/sh intershop2
```

Comment:
Is `intershop2` intended or a typo? If its intended, what is the reason for this change?

Reply:
This change is intended because in line 43 there's already a user added which typically is named _intershop_. I don't know how this could have worked before.